packages/dds/merge-tree/src/test/obliterate.spec.ts

```diff
@@ -240,4 +240,53 @@ describe("obliterate", () => {
 			});
 		});
 	});
+
+	describe.skip("boundary cases", () => {
+		it("obliterate, then insert at the end of the string", () => {
+			obliterateRange({
+				mergeTree: client.mergeTree,
+				start: 5,
+				end: "hello world".length,
+				refSeq,
+				clientId: remoteClientId,
+				seq: refSeq + 1,
+				overwrite: false,
```

Comment:
a bit unrelated, but i'm not sure why overwrite is exposed here. can you take an action item to see if it can be removed from basically all the places that expose it? it should really only exist within the merge tree's obliterate range function

Reply:
AB#15995 for tracking